doc/riverctl.1.scd

```diff
@@ -407,62 +407,70 @@ The _input_ command can be used to create a configuration rule for an input
 device identified by its _name_.
 The _name_ of an input device consists of its type, its numerical vendor id,
 its numerical product id and finally its self-advertised name, separated by -.
+A _glob_ pattern can be used in place of _name_ to match input devices with less
+specificity. A _glob_ is a string that may optionally have an _\*_ at the
+beginning and/or end. An _\*_ in a _glob_ matches zero or more arbitrary
+characters in the app-id or title.
+
+For example, _abc_ is matched by _a\*_, _\*a\*_, _\*b\*_, _\*c_, _abc_, and
+_\*_ but not matched by _\*a_, _b\*_, _\*b_, _c\*_, or _ab_. Note that _\*_
+matches everything while _\*\*_ and the empty string are invalid.
 
 A list of all device properties that can be configured may be found below.
 However note that not every input device supports every property.
 
-*input* _name_ *events* *enabled*|*disabled*|*disabled-on-external-mouse*
+*input* _name_|_glob_ *events* *enabled*|*disabled*|*disabled-on-external-mouse*
```

Comment:
From my point of view, an name is just a glob pattern that matches exactly one input.
```suggestion
*input* _glob_ *events* *enabled*|*disabled*|*disabled-on-external-mouse*
```